recipes/libsigcpp/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
+  "2.10.4":
+    folder: "2.x.x"
   "3.0.0":
     folder: "3.x.x"
```

Comment:
We tend to avoid adding older versions, can you please confirm you need this exact release?

Reply:
You think I would spend time on old software if it wasn't necessary? ;-)

Glibmm needs 2.x (I tested with the 3.x package and that did not work). Linux distro's seem to still package 2.10.4 too (even the ones like Arch Linux, who tend to be on the bleeding edge). So there seems to be good reason to do so.